src/components/Product.jsx

```diff
@@ -0,0 +1,11 @@
+function Product(props) {
+    return (
```

Comment:
Het is netter om je props al op regel 1  tussen () te schrijven zodat je later niet overal props hoeft te schrijven, dan hoef je op andere regels alleen de namen tussen accolades te zetten zonder props ervoor.
De code komt er zo uit te zien :

function Product({textLabel, imageSrc, imageAlt, nameBag, price}) {
  return (

   <article>
            <span>{textLabel}</span>
            <img src={imageSrc} alt={imageAlt}/>
            <p>{nameBag}</p>
            <h4>{price}</h4>
    </article>
);

}